tests/Kernel/Paragraphs/TimelineParagraphTest.php

```diff
@@ -0,0 +1,117 @@
+    $this->assertEquals('Description 2', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item:nth-child(2) div.ecl-timeline2__content')->text()));
+    $this->assertEquals('Label 3', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item:nth-child(3) div.ecl-timeline2__label')->text()));
+    $this->assertEquals('Title 3', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item:nth-child(3) div.ecl-timeline2__title')->text()));
+    $this->assertEquals('Description 3', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item:nth-child(3) div.ecl-timeline2__content')->text()));
+    $this->assertEquals('Label 4', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(4) div.ecl-timeline2__label')->text()));
+    $this->assertEquals('Title 4', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(4) div.ecl-timeline2__title')->text()));
+    $this->assertEquals('Description 4', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(4) div.ecl-timeline2__content')->text()));
+    $this->assertEquals('Label 5', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(5) div.ecl-timeline2__label')->text()));
+    $this->assertEquals('Title 5', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(5) div.ecl-timeline2__title')->text()));
+    $this->assertEquals('Description 5', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(5) div.ecl-timeline2__content')->text()));
+    $this->assertEquals('Label 6', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(6) div.ecl-timeline2__label')->text()));
+    $this->assertEquals('Title 6', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(6) div.ecl-timeline2__title')->text()));
+    $this->assertEquals('Description 6', trim($crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed:nth-child(6) div.ecl-timeline2__content')->text()));
+    $this->assertEquals('Show 3 more items', trim($crawler->filter('button.ecl-button.ecl-button--secondary.ecl-timeline2__toggle span.ecl-button__container span.ecl-button__label')->text()));
+
+    $paragraph->set('field_oe_timeline_expand', '6');
+    $paragraph->save();
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+
+    $this->assertCount(6, $crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item'));
+    $this->assertCount(0, $crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--collapsed'));
+    $this->assertCount(0, $crawler->filter('ol.ecl-timeline2 li.ecl-timeline2__item.ecl-timeline2__item--toggle'));
```

Comment:
We should test when the button is not collapsed and it stays on the page with the text "Hide 3 items". Not only changing the backend to show everything